CONTRIBUTING.md

```diff
@@ -23,6 +23,10 @@ You can clone this repo and activate it like a normal WordPress plugin, but you'
 * [Composer](https://getcomposer.org/)
 * [Node](https://nodejs.org/)
 
+To run the tests, you'll also need:
+
+* [Docker Desktop](https://www.docker.com/desktop) running Docker Compose version 2.20 or higher
```

Comment:
What if someone has only the daemon and client?

I feel I am the only ~~weirdo~~ one not using GUI tool.

Reply:
Telling people to install Docker Desktop is just the easiest way to get set up, especially for users with little or no experience with containers. If you know what you're doing then you probably know you can use a daemon and client instead of Docker Desktop.

Reply:
> for users with little or no experience

Yes.
I am totally unable to talk to them about work.